app/src/main/java/com/example/dine_and_donate/HomeFragments/NotificationsFragment.java

```diff
@@ -85,4 +86,12 @@ public void onCancelled(@NonNull DatabaseError databaseError) {
             }
         });
     }
+
+    public void setLat(double lat){
```

Comment:
I was using this method so that when I clicked on a notification, it would take me to the Map and zoom in at the specific latitude/longitude of the event.
Deleting it for now.